Makefile

```diff
@@ -147,7 +147,7 @@ endif
 .PHONY: golangci
 golangci:   ## Install golangci
 ifeq (, $(shell which golangci-lint))
-	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s  -- -b $(shell $(GO) env GOPATH)/bin $(GOLANGCI_VERSION)
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh $(GOLANGCI_VERSION) | sh -s -- -b $(shell $(GO) env GOPATH)/bin $(GOLANGCI_VERSION)
```

Comment:
The added `$(GOLANGCI_VERSION)` after `install.sh` and before pipe is not the correct way of passing this and has to be removed. Everything before pipe here is a curl command, and we already have the tag passed specified at the end of this line.

Reply:
Thanks @khos2ow! My understanding on the usage of `sh` there was a bit off. Force pushed the requested changes 🙏